libnavigation-base/src/main/java/com/mapbox/navigation/base/extensions/RouteOptionsEx.kt

```diff
@@ -36,3 +40,44 @@ fun RouteOptions?.supportsRouteRefresh(): Boolean {
     } ?: false
     return isTrafficProfile && isOverviewFull && hasCongestionOrMaxSpeed
 }
+
+/**
+ * Applies default [RouteOptions] parameters to the RouteOptions builder
+ *
+ * @receiver RouteOptions.Builder
+ * @return RouteOptions.Builder
+ */
+fun RouteOptions.Builder.applyDefaultOptions(): RouteOptions.Builder = also {
```

Comment:
Maybe? 🤔 

```suggestion
fun RouteOptions.Builder.applyDefaultRequiredOptions(): RouteOptions.Builder = also {
```

Reply:
These are required options, should we add them implicitly downstream? Clients would need to explicitly call this to avoid `RouteOptions` `IllegalArgumentException`s and we should probably default to these internally to prevent these crashes and not force developers to remember calling it.